examples/benchmark_files/create_benchmark_files.py

```diff
@@ -0,0 +1,100 @@
+import coconut
+from coconut import tools
+
+import json
+import os
+import shutil
+import subprocess
+from shutil import copy2
+from os.path import join
+
+
+"""
+This file creates benchmark pickle files used by the script 'evaluate_examples.py'.
+
+Running this file creates pickle files for the examples in the dictionary 'examples'.
+The key is the folder name of the example and the corresponding value is an int indicating the number of 
+time steps the example should be run.
+In order to exclude some examples, comment out the corresponding key-value pair in the dictionary 'examples'.
+
+To include another example, simply add a new key-value pair to the dictionary 'examples'.
+"""
+
+
+# set paths
+execute_path = os.path.realpath(os.path.dirname(__file__))
+tmp_path = join(execute_path, 'tmp')
+examples_path = os.path.realpath(join(os.path.dirname(__file__), '..'))
+bench_mark = join(examples_path, 'benchmark_files')
+
+# example and tuple with number of time steps and optional additional files
+# comment out examples that you do not want to make benchmark files for
+examples = {
+    'tube_fluent2d_abaqus2d': 2,
+    'tube_fluent2d_abaqus2d_steady': 1,
+    'tube_fluent2d_tube_structure': 2,
+    'tube_fluent3d_abaqus2d': 2,
+    'tube_fluent3d_abaqus3d': 2,
+    'tube_fluent3d_kratos_structure3d': 2,
+    'tube_openfoam3d_abaqus3d': 2,
+    'tube_openfoam3d_kratos_structure3d': 2,
+    'tube_tube_flow_abaqus2d': 2,
+    'tube_tube_flow_tube_ringmodel': 100,
+    'tube_tube_flow_tube_structure': 100,
+    'turek_fluent2d_abaqus2d': 2,
+    'turek_fluent2d_abaqus2d_steady': 1
+    }
+
+
+def set_up():
+    # if os.path.exists(tmp_path):
+    #     shutil.rmtree(tmp_path)
+    os.mkdir(tmp_path)
+    cmd = f'cp -r {join(examples_path, "setup_files")} {join(tmp_path, "setup_files")}'
+    p = subprocess.Popen(cmd, executable='/bin/bash', cwd=tmp_path, shell=True)
+    p.wait()
+
+
+def clean_up():
+    shutil.rmtree(tmp_path)
+
+
+def create_benchmark(example, number_of_timesteps):
+    # set paths
+    tmp_example_path = join(tmp_path, example)
+
+    # copy example folder to tmp
+    os.mkdir(tmp_example_path)
+    for file in ('parameters.json', 'setup.py'):
+        copy2(join(examples_path, example, file), tmp_example_path)
```

Comment:
why use copy2 instead of copy?

Reply:
`copy2` allows the destination to be a directory (instead of the complete target filename). I have started to use copy2 by default because it has more capabilities (for the difference see [this post](https://stackoverflow.com/questions/123198/how-can-a-file-be-copied))

Reply:
Since `copy` does the same thing, except for copying metadata, I have changed it to `copy(src, dest)`